install/kubernetes/cilium/values.yaml

```diff
@@ -642,6 +642,19 @@ enableK8sEventHandover: false
 # -- Enable CiliumEndpointSlice feature.
 enableCiliumEndpointSlice: false
 
+envoyConfig:
```

Comment:
you need to put this block in values.yaml.tmp file as well ?

Reply:
You tell me :-) Do I?

Reply:
haha sorry about wrong question mark usage.

Recently, values.yaml.tmpl is the source of truth, and values.yaml is actually generated from values.yaml.tmpl file.

Reply:
Done :-)